src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/web/LoginController.java

```diff
@@ -0,0 +1,30 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.web;
+
+import lombok.extern.java.Log;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Log
+@Controller
+public class LoginController {
+
+  @RequestMapping("/login")
+  public String login() {
+    log.info("login - page called");
+    return "login";

```

Comment:
whats the purpose of this string? explain in comment

Reply:
done